.github/workflows/pst-amt-sock-kickoff.yml

```diff
@@ -2,13 +2,46 @@ name: SOCKS Issue creation kickoff Workflow
 on:
   issues:
     types: [opened, edited, reopened]
-    
+
 env:
   ISSUE_ID: ${{ github.event.issue.number }}
 
 jobs:
+  parse_ssh_key:
+    runs-on: ubuntu-latest
+    outputs:
+      ssh_key: ${{ fromJSON(steps.parse.outputs.payload)[' Public SSH Key'] }}
+    steps:
+      - name: Run Issue form parser
+        id: parse
+        uses: department-of-veterans-affairs/issue-forms-body-parser@main
+        with:
+          issue_id: ${{ github.event.inputs.issue_number }}
+          repository: ${{ env.GH_REPO_FOR_ISSUES }}
+          separator: '###'
+          label_marker_start: " "
+          label_marker_end: "\n"
+  validate_ssh_key:
+    runs-on: ubuntu-latest
+    needs: parse_ssh_key
+    env:
+      PUBLIC_KEY: "${{ needs.parse_ssh_key.outputs.ssh_key }}"
+    if: |
+      needs.parse_ssh_key.outputs.ssh_key != ''
+    steps:
+      - name: Run ssh public key validation
+        id: fingerprint
+        shell: bash
+        run: ssh-keygen -lf /dev/stdin <<< "$PUBLIC_KEY"
+      - name: Notify failure
+        id: ssh_key_invalid
+        if: ${{ failure() }}
+        shell: bash
+        run: "echo \"::error::You seem to have uploaded an invalid ssh public key: $PUBLIC_KEY\""
```

Comment:
run: echo "::error:: You seem to have uploaded an invalid ssh public key: $PUBLIC_KEY"
        Above will work as well rather than having all those escape character